test/blackbox-tests/test-cases/dune-cache/cache-man.t

```diff
@@ -91,10 +91,13 @@ Testing the output of dune cache trim.
   
   OPTIONS
          --size=BYTES
-             Size to trim the cache to.
+             Size to trim the cache to. BYTES is the number of bytes followed
+             by a unit. Byte units can be one of B, bytes, kB, KB, kilobytes,
+             KiB, KiB, kibibytes, MB, megabytes, MiB, mebibytes, GB, gigabytes,
+             GiB, gibibytes, TB, terabytes, TiB or tebibytes.
```

Comment:
To be honest, this help message seems pretty excessive. Noone is going to run the trimmer with `--size=1megabytes` (which, by the way, isn't even correct and we don't have a singular form).

Reply:
I agree it is excessive. Where is a better place to document what we accept? Or are you suggesting we remove the extra quantifiers?

Reply:
I've shortened it to just the main units now. I've updated the PR description so it is more clear too.